c2corg_ui/static/js/map/map.js

```diff
@@ -313,7 +321,10 @@ app.MapController.DEFAULT_POINT_ZOOM = 12;
  */
 app.MapController.prototype.recenterOnExtent_ = function(extent, options) {
   var mapSize = this.map.getSize();
-  if (mapSize) {
+  if (!mapSize || !ol.extent.getWidth(extent) || !ol.extent.getHeight(extent)) {
```

Comment:
[ol.extent.isEmpty](http://openlayers.org/en/master/apidoc/ol.extent.html#.isEmpty) might be useful.